docs/architecture/porting-existing-aspnet-apps/app-startup-differences.md

```diff
@@ -7,8 +7,63 @@ ms.date: 11/13/2020
+
+Many NuGet packages for ASP.NET MVC and Web API use the [WebActivator](https://github.com/davidebbo/WebActivator) package to let them run some code during application startup. By convention, this code would typically be added to an `App_Start` folder and would be configured via attribute to run either immediately before or just after `Application_Start`.
+
+It's also possible to use the [Open Web Interface for .NET (OWIN) and Project Katana with ASP.NET MVC](https://docs.microsoft.com/aspnet/aspnet/overview/owin-and-katana/getting-started-with-owin-and-katana). When doing so, the application will include a `Startup.cs` file that is responsible for setting up request middleware in a way that's very similar to how ASP.NET Core behaves.
+
+If you need to run code when your ASP.NET MVC app starts up, it will typically use one of these approaches.
+
+## ASP.NET Core Startup
+
+As noted previously, ASP.NET Core applications are standalone programs. As such, they typically include a `Program.cs` file containing the entry point for the application. A typical such file is shown in Figure 2-1.
+
+```csharp
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        CreateHostBuilder(args).Build().Run();
+    }
+
+    public static IHostBuilder CreateHostBuilder(string[] args) =>
+        Host.CreateDefaultBuilder(args)
+            .ConfigureWebHostDefaults(webBuilder =>
+            {
+                webBuilder.UseStartup<Startup>();
+            });
+}
+```
+
+**Figure 2-1. A typical ASP.NET Core Program.cs file.
+
+The code shown in Figure 2-1 creates a *host* for the application, builds it, and runs it. The ASP.NET Core application runs within the host configured by the `IHostBuilder` shown. While it's possible to completely configure an ASP.NET Core app using the `IHostBuilder`, typically the bulk of this work is done in a `Startup` class.
+
+The `Startup` class exposes two methods to the host, `ConfigureServices` and `Configure`. The `ConfigureServices` method is used to define the services the app will use and their respective lifetimes. The `Configure` method is used to define how each request to the app will be handled by setting up a request pipeline composed of middleware.
+
+Code that isn't related to configuring the application's services or request pipeline, but which you need to run when the app begins, is typically placed in `Program.cs` or registered as an `IHostedService` which will be started by the [generic host](https://docs.microsoft.com/aspnet/core/fundamentals/host/generic-host?view=aspnetcore-3.1) when the application starts.
```

Comment:
```suggestion
Code that isn't related to configuring the application's services or request pipeline, but that you need to run when the app begins, it's typically placed in `Program.cs` or registered as an `IHostedService`, which will be started by the [generic host](https://docs.microsoft.com/aspnet/core/fundamentals/host/generic-host?view=aspnetcore-3.1) when the application starts.
```

Reply:
I'm fine with "but that you need" instead of "but which you need" but it doesn't make sense to say "Code... it's typically placed" rather than "Code... is typically placed".

I also suspect that whole sentence could probably be 2-3 sentences so I'll work on that. :)